docs/overview/renderers/nema_gfx.rst

```diff
@@ -0,0 +1,73 @@
+.. _stm32_nema_gfx:
+
+=============================================
+STM32 NemaGFX GPU (AKA NeoChrom or Chrom-ART)
+=============================================
+
+Some of the more powerful STM32 MCUs such as the
+STM32U5 feature a 2D GPU which can natively draw most
```

Comment:
Small update here, NemaP and NemaPVG are 2.5 D GPUs, as they also provide projections functionality

Reply:
```suggestion
STM32U5 feature a 2D.5 GPU which can natively draw most
```

Like this?

Reply:
2.5 D will do the job. Thank you